docs/source/nlp/language_modeling.mdx

```diff
@@ -0,0 +1,379 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Language modeling
+
+Language modeling fits a model to a corpus. There are two forms of language modeling:
```

Comment:
Not super fond of this sentence. It's the task of predicting words in a sentence in a general.